var/spack/repos/builtin/packages/proj/package.py

```diff
@@ -94,6 +94,10 @@ class Proj(CMakePackage, AutotoolsPackage):
         # patch replaces the TIFF_LIBRARY variable (no longer used) with TIFF_LIBRARIES
         patch("proj-8.1-cmake-3.29-new-tiff-interface.patch", when="+tiff @8:9.1.0 ^cmake@3.19:")
         patch("proj-7-cmake-3.29-new-tiff-interface.patch", when="+tiff @7 ^cmake@3.19:")
+        # tiff does not set TIFF_INCLUDE_DIR tested by proj
+        # version 9.2 contains this patch:
+        # https://github.com/OSGeo/PROJ/blob/9.2/src/lib_proj.cmake#L458
+        patch("tiff_target.patch", when="+tiff @8.1:9.1.1 ^cmake@3.28:")
```

Comment:
The patch on line 95 of this file already covers this exact range of changes and because of the open ended constraint, is also applied when this is applied, so this patch no longer works. We need to make the constraints above tighter.

Reply:
Indeed. Good catch! I pushed a patch.